### DISCUSSION TITLE
2022-05-03 Weekly Meeting 54 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#223] [Weekly Meetings]
Weekly Meeting 54 is happening on Wednesday May 3rd, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
- Schedule of leadership meeting set to Friday 12PM - 1PM PST

Working Group Updates:
- OMI Website update - blog update
- OMI-Chan group
- Scripting group
- glTF Extensions Group
- https://github.com/omigroup/media
- any others?

Demos? / Topics?:
- @madjin showed off https://3d.m3org.com/

**Comment by humbletim** [comment]
just to mention: Proposing UX Research Working Group #222

**Reply by mrmetaverse** [reply]
thank you @humbletim I came here specifically to make sure it was discussed.

**Reply by robertlong** [reply]
We discussed it, I left a comment here: https://github.com/omigroup/omigroup/issues/222

**Reply by mrmetaverse** [reply]
thanks @robertlong!

**Comment by mrmetaverse** [comment]
Created webhooks for our working groups with github repos to make it easier to track across.
